### ISSUE TITLE
🐞 project-origin-wallet package installation fails : sqlServer not supported

### ISSUE THREAD

**Issue by InassBelgcir** [project-origin/wallet#56]
### Describe the Bug

Wallet Installation fails 

**Error: INSTALLATION FAILED: unable to build kubernetes objects from release manifest: resource mapping not found for name: "cnpg-wallet-db" namespace: "default" from "": no matches for kind "Cluster" in version "postgresql.cnpg.io/v1"
ensure CRDs are installed first**

### To Reproduce

```bash
1.connect to the cluster
2.run helm repo add project-origin https://project-origin.github.io/helm-registry (successful)
3.helm install project-origin-wallet project-origin/project-origin-wallet --version 0.1.2 --set "wallet.externalUrl=https://spider-projects.net"
```

### Expected Behavior

Installation success

### Screenshots

![image](https://github.com/project-origin/wallet/assets/141126356/cf4f09ed-5d25-492c-9eb0-bcf8c34b39cc)

### Desktop

- OS: Windows 10
- Browser: Edge

**Comment by MartinSchmidt** [comment]
This should be solved by the new feature in issue #65, it relates to the current chart requiring CloudNativePG

**Comment by MartinSchmidt** [comment]
This bug will be closed, as it currently is the implemented functionality, that it only works with CloudNativePG. #65 for future "bring you own database"
